src/WebJobs.Script/Description/FunctionInvokerBase.cs

```diff
@@ -81,7 +85,99 @@ protected bool InitializeFileWatcherIfEnabled()
             return false;
         }
 
-        public abstract Task Invoke(object[] parameters);
+        public Task Invoke(object[] parameters)
```

Comment:
Yes! Very nice to see this pushed down to the base.
We should be able to take this a step further and change the `InvokeInternal` to take the system parameters we extract so we don't have to do that again in the invokers (e.g. `ExecutionContext`, `CancellationToken`, `TraceWriter`, etc.), perhaps rename that to something like `InvokeFunction` as well?
So we'd have that method signature be something like `InvokeFunction(object[] arguments, ExecutionContext context, TraceWriter traceWriter, ...)`